README.md

```diff
@@ -82,6 +82,9 @@ as well as the following command line options:
 * `-i/--in-place`: Overwrites the input files when formatting instead of
   printing the results to standard output.
 
+* `-p/--parallel`: Whether or not to run formatting or linting in parallel
+  consuming all possible resources printing the results to standard output.
```

Comment:
Can you update this the same way you updated the flag text in the code to sound less dire? 😄 

Something like "[...] to run formatting or linting in parallel across multiple cores".

Also, we don't need to say anything about stdout, right? If this option is paired with `-i`, the files will still be updated in-place?

Reply:
Yep sorry, bad copy paste